homeassistant/components/lamarzocco/switch.py

```diff
@@ -0,0 +1,92 @@
+"""Switch platform for La Marzocco espresso machines."""
+from collections.abc import Callable, Coroutine
+from dataclasses import dataclass
+from typing import Any
+
+from homeassistant.components.switch import SwitchEntity, SwitchEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .coordinator import LaMarzoccoUpdateCoordinator
+from .entity import LaMarzoccoEntity, LaMarzoccoEntityDescription
+
+
+@dataclass(frozen=True, kw_only=True)
+class LaMarzoccoSwitchEntityDescription(
+    LaMarzoccoEntityDescription,
+    SwitchEntityDescription,
+):
+    """Description of a La Marzocco Switch."""
+
+    control_fn: Callable[[LaMarzoccoUpdateCoordinator, bool], Coroutine[Any, Any, bool]]
+    is_on_fn: Callable[[LaMarzoccoUpdateCoordinator], bool]
```

Comment:
Currently you are only using the LaMarzoccoClient from the coordinator, you could directly pass it in instead

Reply:
as discussed: will need coordinator here once Bluetooth comes back in and don't want to go back and forth